docs/concepts/whatisvpp/whatisvpp.rst

```diff
@@ -0,0 +1,363 @@
+
+.. figure:: /_images/VPP_sample_data_plane_management_agent_x260.jpg
+   :alt: VPP Using ODL Honeycomb as a Data Plane Management Agent
+
+   VPP Using ODL Honeycomb as a Data Plane Management Agent
+
+This would be one way to implement Bridge Domains.
+
+Primary Characteristics Of VPP
+------------------------------
+
+Improved fault-tolerance and ISSU
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Improved fault-tolerance and ISSU when compared to running similar
+packet processing in the kernel:
+
+* crashes seldom require more than a process restart
+* software updates do not require system reboots
+* development environment is easier to use and perform debug than similar kernel code
+* user-space debug tools (gdb, valgrind, wireshark)
+* leverages widely-available kernel modules (uio, igb_uio): DMA-safe memory
+
+Runs as a Linux user-space process:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* same image works in a VM, in a Linux container, or over a host kernel
+* KVM and ESXi: NICs via PCI direct-map
+* Vhost-user, netmap, virtio paravirtualized NICs
+* Tun/tap drivers
+* DPDK poll-mode device drivers
+
+Integrated with the DPDK, VPP supports existing NIC devices including:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Intel i40e, Intel ixgbe physical and virtual functions, Intel e1000, virtio, vhost-user, Linux TAP
+* HP rebranded Intel Niantic MAC/PHY
+* Cisco VIC
+
+Security issues considered:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Extensive white-box testing by Cisco's security team
+* Image segment base address randomization
+* Shared-memory segment base address randomization
+* Stack bounds checking
+* Debug CLI "chroot"
+
+The vector method of packet processing has been proven as the primary
+punt/inject path on major architectures.
+
+Supported Architectures
+-----------------------
+.. list-table:: Supported Architectures
+   :widths: 50
+   :header-rows: 1
+
+   * - The VPP platform supports:
+
+   * - x86/64
+     
+Supported Packaging Models
+--------------------------
+
+The VPP platform supports package installation on the following
+operating systems:
+
+.. list-table:: Supported Packaging Models
+   :widths: 50 
+   :header-rows: 1
+
+   * - Operating System:
+
+   * - Debian
+   * - Ubuntu 16.04
+   * - CentOS 7.3
+
+
+Performance Expectations
+------------------------
+
+One of the benefits of this implementation of VPP is its high
+performance on relatively low-power computing. This high level of
+performance is based on the following highlights:
+
+* High-performance user-space network stack for commodity hardware
+* The same code for host, VMs, Linux containers
+* Integrated vhost-user virtio backend for high speed VM-to-VM connectivity
+* L2 and L3 functionality, multiple encapsulations
+* Leverages best-of-breed open source driver technology: DPDK
+* Extensible by use of plugins
+* Control-plane / orchestration-plane via standards-based APIs
+
+Performance Metrics
+-------------------
```

Comment:
You need to ditch these dated performance figures and refer people to relevant sections of the most recent FD.io CSIT performance report. https://docs.fd.io/csit/rls1804/report/

Reply:
Would this be a link to the link?  Or reuse some sections/abridge them specifically for this page?